src/slices/connectionsSlice.ts

```diff
@@ -5,47 +5,34 @@ const connectionsSlice = createSlice({
     name: "connections",
     initialState: {
         websocket: { name: "Backend WebSocket", isConnected: false },
-        boards: [] as Connection[],
+        boards: {} as Record<string, Connection>,
```

Comment:
this is specified on the document as being an array, there was a bug in the backend where it would send it as a map, it should be fixed there, not here, unless it is better to do it this way.

If using a map is better I would change the type signature to `Record<string, bool>`, where the boolean is weather the board is connected or not

Reply:
Okay, then let's do it as an array